src/main/java/org/commcare/formplayer/services/ScheduledTasks.java

```diff
@@ -0,0 +1,48 @@
+package org.commcare.formplayer.services;
+
+import com.timgroup.statsd.StatsDClient;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.commcare.formplayer.Application;
+import org.commcare.formplayer.repo.FormSessionRepo;
+import org.commcare.formplayer.util.Constants;
+import org.commcare.formplayer.util.RequestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+
+@Component
+public class ScheduledTasks {
+
+    private final Log log = LogFactory.getLog(ScheduledTasks.class);
+
+    private static final SimpleDateFormat dateFormat = new SimpleDateFormat("HH:mm:ss");
+
+    @Autowired
+    protected FormSessionRepo formSessionRepo;
+
+    @Autowired
+    private StatsDClient datadogStatsDClient;
+
+    // the default "-" corresponds to Scheduled.CRON_DISABLED
+    @Scheduled(cron= "${commcare.formplayer.scheduledTasks.purge.cron:-}")
```

Comment:
If you are providing a default here, you won't also need to replicate in `application.properties` as far as I'm aware (and it might be confusing for someone who changes these defaults and misses that the app is also overriding them in the default config)

Reply:
Ahh, I didn't know that `resources/application.properties` were the defaults that are included jar. I'll remove those defaults because having the defaults be in code seem much safer, and the default is that this will not run (cron expression `-`).